terraso_backend/apps/project_management/migrations/0028_site_elevation.py

```diff
@@ -0,0 +1,34 @@
+# Copyright © 2024 Technology Matters
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see https://www.gnu.org/licenses/.
+
+# Generated by Django 5.0.4 on 2024-05-07 18:52
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("project_management", "0027_project_measurement_units"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="site",
+            name="elevation",
+            field=models.FloatField(default=0.0),
```

Comment:
question: is this migration in sync with the code? i didn't expect to see the `default=0.0` here?